webcompat/db/__init__.py

```diff
@@ -46,23 +40,3 @@ def __init__(self, issue_id, summary, url, body):
         self.body = body
 
 IssueBase.metadata.create_all(bind=issue_engine)
```

Comment:
nit: two blank lines after a class (or above `IssueBase.metadata.…` in this case) ;)